wormhole-connect/src/utils/confirmBeforeLeaving.ts

```diff
@@ -1,24 +0,0 @@
-import { useEffect } from 'react';
-
-const unloadHandler = (e: Event) => {
-  e = e ?? window.event;
-  if (e) e.returnValue = true;
-  return true;
-};
-
-const useConfirmBeforeLeaving = (shouldConfirm: boolean) => {
```

Comment:
This seems like a useful hook. Should we keep it and use it when a transfer is being performed? cc @artursapek

Reply:
I've added it back under hooks.